[INFO] Central web server started at [2025-03-30 21:25:15.376]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-30 21:25:15.508]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:25:23.851]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:25:23.911]
[ERROR] No valid dictionary hash files found while trying to  at [2025-03-30 21:25:23.914]
[ERROR] Failed to synchronize dictionary with server http://192.168.0.19:5099/api/central/check-connection: No valid dictionary hash files found. at [2025-03-30 21:25:23.915]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: No valid dictionary hash files found. at [2025-03-30 21:25:23.918]
[ERROR] Cannot connect to calculating server due to: No valid dictionary hash files found. at [2025-03-30 21:25:23.920]
[INFO] Central web server started at [2025-03-30 21:28:13.435]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-30 21:28:13.535]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:28:28.057]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:28:28.100]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:28:28.105]
[ERROR] Cannot connect to calculating server due to: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:28:28.106]
[INFO] Central web server started at [2025-03-30 21:32:05.102]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-30 21:32:05.209]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:32:13.675]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:32:13.710]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:32:13.718]
[ERROR] Cannot connect to calculating server due to: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:32:13.719]
[INFO] Made request to synchronize dictionary at [2025-03-30 21:33:24.317]
[ERROR] Error while synchronizing dictionary: Incorrect Content-Type:  at [2025-03-30 21:33:24.321]
[INFO] Made request to synchronize dictionary at [2025-03-30 21:34:03.736]
[ERROR] Received dictionary pack was invalid while trying to synchronize dictionary at [2025-03-30 21:34:03.738]
[ERROR] Error while synchronizing dictionary: Invalid dictionary pack content or format. at [2025-03-30 21:34:03.739]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:35:22.422]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:35:22.476]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:35:22.477]
[ERROR] Cannot connect to calculating server due to: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:35:22.478]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:36:09.123]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:36:09.167]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:36:09.171]
[ERROR] Cannot connect to calculating server due to: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:36:09.171]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:36:19.370]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:36:19.413]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:36:19.415]
[ERROR] Cannot connect to calculating server due to: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:36:19.416]
[INFO] Central web server started at [2025-03-30 21:36:44.273]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-30 21:36:44.380]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:36:55.504]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:36:55.543]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:36:55.547]
[ERROR] Cannot connect to calculating server due to: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:36:55.547]
[INFO] Central web server started at [2025-03-30 21:38:53.924]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-30 21:38:54.018]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:39:00.103]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:39:00.141]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:39:00.145]
[ERROR] Cannot connect to calculating server due to: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:39:00.146]
[INFO] Central web server started at [2025-03-30 21:41:21.479]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-30 21:41:21.582]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:41:28.625]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:41:28.661]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:41:28.676]
[ERROR] Cannot connect to calculating server due to: The value cannot be an empty string. (Parameter 'path') at [2025-03-30 21:41:28.677]
[INFO] Central web server started at [2025-03-30 21:43:10.836]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-30 21:43:10.933]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:43:16.755]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:43:16.794]
[ERROR] Failed to synchronize dictionary with server 192.168.0.19: An invalid request URI was provided. Either the request URI must be an absolute URI or BaseAddress must be set. at [2025-03-30 21:43:16.802]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: An invalid request URI was provided. Either the request URI must be an absolute URI or BaseAddress must be set. at [2025-03-30 21:43:16.804]
[ERROR] Cannot connect to calculating server due to: An invalid request URI was provided. Either the request URI must be an absolute URI or BaseAddress must be set. at [2025-03-30 21:43:16.805]
[INFO] Central web server started at [2025-03-30 21:44:26.766]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-30 21:44:26.856]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:44:33.354]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:44:33.391]
[ERROR] Failed to synchronize dictionary with server 192.168.0.19: Connection refused (192.168.0.19:80) at [2025-03-30 21:44:33.399]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: Connection refused (192.168.0.19:80) at [2025-03-30 21:44:33.402]
[ERROR] Cannot connect to calculating server due to: Connection refused (192.168.0.19:80) at [2025-03-30 21:44:33.402]
[INFO] Central web server started at [2025-03-30 21:45:19.717]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-30 21:45:19.808]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:45:28.548]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:45:28.588]
[ERROR] Failed to synchronize dictionary with server 192.168.0.19: Server 192.168.0.19 responded with status code InternalServerError at [2025-03-30 21:45:30.734]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: Server 192.168.0.19 responded with status code InternalServerError at [2025-03-30 21:45:30.738]
[ERROR] Cannot connect to calculating server due to: Server 192.168.0.19 responded with status code InternalServerError at [2025-03-30 21:45:30.738]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:46:10.695]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:46:10.735]
[INFO] Successfully synchronized dictionary with server 192.168.0.19 at [2025-03-30 21:46:14.756]
[INFO] Calculating server with IP address: 192.168.0.19 successfully connected to the central server at [2025-03-30 21:46:14.773]
[INFO] Central web server started at [2025-03-30 21:46:41.461]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-30 21:46:41.542]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:46:49.885]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:46:49.925]
[INFO] Successfully synchronized dictionary with server 192.168.0.19 at [2025-03-30 21:46:53.259]
[INFO] Calculating server with IP address: 192.168.0.19 successfully connected to the central server at [2025-03-30 21:46:53.272]
[INFO] Made request to synchronize dictionary at [2025-03-30 21:47:45.900]
[INFO] Successfully saved new dictionary at [2025-03-30 21:47:45.974]
[INFO] Successfully synchronized dictionary with server 192.168.0.19 at [2025-03-30 21:47:46.150]
[INFO] Made request to crack password using dictionary method at [2025-03-30 21:47:46.183]
[INFO] Final cracking time with dictionary method was: 01:34:10 at [2025-03-30 21:47:46.187]
[INFO] Central web server started at [2025-03-30 21:50:29.328]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-30 21:50:29.415]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:51:03.573]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:51:03.613]
[ERROR] Failed to synchronize dictionary with server 192.168.0.19: Server 192.168.0.19 responded with status code InternalServerError at [2025-03-30 21:51:07.450]
[ERROR] Failed to connect with calculating server 192.168.0.19 for reason: Server 192.168.0.19 responded with status code InternalServerError at [2025-03-30 21:51:07.459]
[ERROR] Cannot connect to calculating server due to: Server 192.168.0.19 responded with status code InternalServerError at [2025-03-30 21:51:07.463]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:52:45.250]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:52:45.296]
[INFO] Successfully synchronized dictionary with server 192.168.0.19 at [2025-03-30 21:52:48.595]
[INFO] Calculating server with IP address: 192.168.0.19 successfully connected to the central server at [2025-03-30 21:52:48.601]
[INFO] Made request to try to connect calculating server from IP address: 192.168.0.19 at [2025-03-30 21:53:19.988]
[INFO] Successfully connected to calculating server 192.168.0.19 at [2025-03-30 21:53:20.021]
[INFO] Successfully synchronized dictionary with server 192.168.0.19 at [2025-03-30 21:53:22.598]
[INFO] Calculating server with IP address: 192.168.0.19 successfully connected to the central server at [2025-03-30 21:53:22.601]
[INFO] Made request to synchronize dictionary at [2025-03-30 21:53:41.857]
[INFO] Successfully saved new dictionary at [2025-03-30 21:53:41.988]
[INFO] Successfully synchronized dictionary with server 192.168.0.19 at [2025-03-30 21:53:42.171]
[INFO] Successfully synchronized dictionary with server 192.168.0.19 at [2025-03-30 21:53:42.304]
[INFO] Made request to crack password using dictionary method at [2025-03-30 21:53:42.337]
[INFO] Final cracking time with dictionary method was: 01:34:10 at [2025-03-30 21:53:42.342]
[INFO] Central web server started at [2025-03-31 06:03:47.834]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-31 06:03:47.998]
[INFO] Made request to try to connect calculating server from IP address: 192.168.80.57 at [2025-03-31 06:03:58.932]
[INFO] Successfully connected to calculating server 192.168.80.57 at [2025-03-31 06:03:58.989]
[INFO] Successfully synchronized dictionary with server 192.168.80.57 at [2025-03-31 06:03:59.210]
[INFO] Calculating server with IP address: 192.168.80.57 successfully connected to the central server at [2025-03-31 06:03:59.215]
[INFO] Central web server started at [2025-03-31 06:07:42.681]
[INFO] Successfully connected to the PostgreSQL database instance at [2025-03-31 06:07:42.796]
[INFO] Made request to try to connect calculating server from IP address: 192.168.80.57 at [2025-03-31 06:07:52.256]
[INFO] Successfully connected to calculating server 192.168.80.57 at [2025-03-31 06:07:52.329]
[INFO] Successfully synchronized dictionary with server 192.168.80.57 at [2025-03-31 06:07:52.491]
[INFO] Calculating server with IP address: 192.168.80.57 successfully connected to the central server at [2025-03-31 06:07:52.495]
[INFO] Made request to synchronize dictionary at [2025-03-31 06:10:45.785]
[INFO] Successfully saved new dictionary at [2025-03-31 06:10:56.820]
[INFO] Successfully synchronized dictionary with server 192.168.80.57 at [2025-03-31 06:11:12.636]
[INFO] Made request to crack password using dictionary method at [2025-03-31 06:11:12.714]
[INFO] Final cracking time with dictionary method was: 01:34:10 at [2025-03-31 06:11:12.723]
[INFO] Made request to crack password using brute force method at [2025-03-31 06:11:27.305]
[INFO] Final cracking time with brute force method was: 03:34:10 at [2025-03-31 06:11:27.308]
[INFO] Made request to get actual dictionary hash at [2025-03-31 06:11:53.040]
[INFO] Made request to get actual dictionary pack file at [2025-03-31 06:11:59.359]
